rcli/autodetect.py

```diff
@@ -70,9 +70,9 @@ def egg_info_writer(cmd, basename, filename):
     config = dict(parser.items('rcli'))
     for k, v in six.iteritems(config):
         if v.lower() in ('y', 'yes', 'true'):
-            config[k] = True
+            config[k] = True  # type: ignore
```

Comment:
What version of `mypy` are you using that makes this necessary? It would be better to change this to `# type: bool` though.

Reply:
```BASH
1515687676 || mammut mherder master[ ]
/home/mherder/rcli RC: 127 $
source .tox/py36/bin/activate
1515687689 || mammut mherder master[ ]
/home/mherder/rcli: 0 $
mypy --version
mypy 0.560
```

Reply:
Changing to type bool...

Reply:
Is that checked in?

Reply:
No it turns out it doesn't work.